Makefile

```diff
@@ -44,6 +44,8 @@ clean:
 	mkdir -p build/locale
 
 locale:
+	# For generating a new file to let locales name localizable
+	python scripts/translate_locales.py
```

Comment:
This script should run every-time a new locale is added. so while adding a new locale and committing, the file will be regenerated.

Yap. seems like this should be a management command